src/zope/browserresource/tests/test_directives.py

```diff
@@ -16,49 +16,57 @@
 
 import os
 import unittest
-try:
-    from cStringIO import StringIO
-except ImportError:
-    from io import StringIO
+from io import StringIO
+
 from zope import component
-from zope.interface import Interface, implements, directlyProvides, providedBy
+from zope.interface import Interface, implementer
 
 import zope.security.management
 from zope.configuration.xmlconfig import xmlconfig, XMLConfig
 from zope.configuration.exceptions import ConfigurationError
 from zope.publisher.browser import TestRequest
-from zope.publisher.interfaces import IDefaultViewName
-from zope.publisher.interfaces.browser import IBrowserPublisher
+
+
 from zope.publisher.interfaces.browser import IBrowserRequest
-from zope.publisher.interfaces.browser import IBrowserSkinType, IDefaultSkin
-from zope.security.proxy import removeSecurityProxy, ProxyFactory
-from zope.security.permission import Permission
-from zope.security.interfaces import IPermission
+
+from zope.security.proxy import ProxyFactory
+
+
```

Comment:
Why are the so many blocks of imports separated by newlines? (sometimes one sometimes two)